test/models/survey_test.exs

```diff
@@ -185,4 +185,16 @@ defmodule Ask.SurveyTest do
       assert expired? == true
     end
   end
+
+  describe "panel survey" do
+    test "creates a survey inside a panel survey" do
```

Comment:
I don't understand what we are asserting here. What we are comparing is the result of the insert, with getting the data from the db (that isn't always the same as what we inserted?)

Reply:
We're just checking that a survey can be assigned to a panel survey. An assertion checking exactly that was missing. I've just added it. Please take a look again.

The comparison between the (factory created) inserted panel survey with the loaded panel survey from the DB is just that. Trying to check that nothing was missing after DB insertion and reading.